miio/vacuumcontainers.py

```diff
@@ -292,6 +292,7 @@ def __init__(self, data: Dict[str, Any]) -> None:
         self.main_brush_total = timedelta(hours=300)
         self.side_brush_total = timedelta(hours=200)
         self.filter_total = timedelta(hours=150)
+        self.sensors_total = timedelta(hours=30)
```

Comment:
Use singular here, as property is "sensor_dirty". Or if you thing it's worth renaming it now, maybe introduce a new, renamed property and `@deprecate` the old one.

Reply:
Thanks for taking the time to have a look. I have fixed the issues.